src/SSLSocket.c

```diff
@@ -325,6 +325,21 @@ The user-defined argument optionally defined by SSL_CTX_set_msg_callback_arg() o
 		content_type, (int)len);
 }
 
+void SSL_CTX_keylog_callback(const SSL *ssl, const char *line)
+{
+	const char* sslkeylogfile = getenv("SSLKEYLOGFILE");
+	if (sslkeylogfile)
+	{
+		FILE *fptr;
+
+		fptr = fopen(sslkeylogfile, "at");
+		if (fptr)
```

Comment:
It would be nice to have an error log if the file failed to open, for example due to a permissions problem
- [ ] Error log if keylog file can't be opened.